### PR TITLE
Prepare v0.4.1 release

### PR DESCRIPTION
**Pull request by raphaelrobert**
This PR prepares release v0.4.1 which fixes the API as mentioned in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 0.4.1 (2022-06-07)
+
 ### Added
- - Signature sub-module of the ciphersuite module is now public.
- - Signature keys can be imported and exported with the crypto-subtle feature.
- - BasicCredentials can now be created from existing signature keys.
+ - [#873:] Signature sub-module of the ciphersuite module is now public.
```
Comment (franziskuskiefer) [inline CHANGELOG.md:12]:
```suggestion
 - [#873:](https://github.com/openmls/openmls/pull/873) Signature sub-module of the ciphersuite module is now public.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,13 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+ - [#873:] Signature keys can be imported and exported with the crypto-subtle feature.
```
Comment (franziskuskiefer) [inline CHANGELOG.md:13]:
```suggestion
 - [#873:](https://github.com/openmls/openmls/pull/873) Signature keys can be imported and exported with the crypto-subtle feature.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,13 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+ - [#873:] BasicCredentials can now be created from existing signature keys.
```
Comment (franziskuskiefer) [inline CHANGELOG.md:14]:
```suggestion
 - [#873:](https://github.com/openmls/openmls/pull/873) BasicCredentials can now be created from existing signature keys.
```

**Review by franziskuskiefer — COMMENTED** [review]
lgtm with the added links

**Review by franziskuskiefer — APPROVED** [review]
thanks!

**Comment by adzialocha** [comment]
Thank you so much!

**Pull request merged**